ccd-definition/FixedLists/CareSupervision/ChildGenderList.json

```diff
@@ -16,8 +16,8 @@
   {
     "LiveFrom": "01/01/2017",
     "ID": "ChildGenderList",
-    "ListElementCode": "They identify in another way",
-    "ListElement": "They identify in another way",
+    "ListElementCode": "Indeterminate",
```

Comment:
I believe this will break existing data - can change the label (ListElement) without changing the ListElementCode, then CCD validation will still pass